apis/python/tests/test_dataframe_io_roundtrips.py

```diff
@@ -201,8 +205,17 @@ def test_adata_io_roundtrips(
     with Experiment.open(ingested_uri) as exp:
         adata1 = to_anndata(exp, "meas", obs_id_name=outgest_default_index_name)
         outgested_obs = adata1.obs
+
     assert_frame_equal(outgested_obs, outgested_df)
 
+    expected = deepcopy(adata0)
+    # Patch in the expected "outgested" DataFrame (which in these test cases is known to differ
+    # from what was ingested).
+    expected.obs = outgested_df
+    # TODO: outgest same format that was ingest
```

Comment:
```suggestion
    # TODO: outgest same format that was ingested
```

Reply:
I fixed something else I was doing in this test, that obviates the need for this comment (and the line it applied to).